docs/sheets/GNSS.md

```diff
@@ -1,236 +1,251 @@
-> 
 > ### :heavy_check_mark: Delta prerequisites
-> 
-> * There can be only one __mark__ attached to a __monument__.
-> * Only one __monument__ can be associated with any given __mark__.
+> * The __receiver__ needs to be matched in the __assets/receiver.csv__ file.
 >
 > ### :small_orange_diamond: Delta checks
->
-> * The "mark type" of the __monument__ must be one of:
-> * * "Shallow Rod / Braced Antenna Mount"
-> * * "Wyatt/Agnew Drilled-Braced"
-> * * "Pillar"
-> * * "Steel Mast"
-> * * "Unknown"
-> * The "ground relationship" must be equal to or less than zero.
+> * A given __receiver__ can only have one __firmware__ version at any given time.
 
-## :three: <a name="antenna"></a>_Installing or replacing a GNSS antenna_
 
-> ### :page_with_curl: Files to update
->
-> * __install/antennas.csv__
+## :two: <a name="receiver"></a>_Installing or replacing a GNSS receiver_
 >
-> ### :page_with_curl: Reference files
->
-> * __assets/antennas.csv__
-> * __network/marks.csv__
->
-> ### :information_source: General requirements
->
-> * GNSS __mark__ code (_where the antenna is mounted_)
->
-> ### :information_source: Field requirements
-> 
-> * Antenna make, model, and serial number.
-> * Installation details, such as height, offset from the mark, and azimuth.
-> * Antenna installation or removal times.
-> 
-> ### :heavy_check_mark: Delta prerequisites
-> 
-> * The antennas need to be listed in the __assets/antennas.csv__ file.
-> * The associated __mark__ needs to be listed in the __network/marks.csv__ file.
-> 
-> ### :small_orange_diamond: Delta checks
-> 
-> * A given __antenna__ can only be installed at a single __mark__ at any given time.
-> * A __mark__ can only have one __antenna__ mounted at any given time.
-
-## :four: <a name="receiver"></a>_Installing or replacing a GNSS receiver_
-
 > ### :page_with_curl: Files to update
->
 > * __install/receivers.csv__
+> * __install/firmware.csv__
 >
 > ### :page_with_curl: Reference files
->
 > * __assets/receivers.csv__
 > * __network/marks.csv__
+> * __install/sessions.csv__
 >
 > ### :information_source: General requirements
->
 > * GNSS __mark__ code (_where the antenna is mounted_)
+> * The "receiver model" must exist in the IGS [rcvr_ant.tab](ftp://igs.org/pub/station/general/rcvr_ant.tab) file
 >
 > ### :information_source: Field requirements
-> 
 > * Receiver make, model, and serial number.
 > * Receiver deployment or removal times.
-> 
+>
 > ### :heavy_check_mark: Delta prerequisites
-> 
 > * Receivers need to be listed in the __assets/receivers.csv__ file.
 > * The associated __mark__ needs to be listed in the __network/marks.csv__ file.
-> 
+>
 > ### :small_orange_diamond: Delta checks
-> 
 > * A given __receiver__ can only be deployed to a single __mark__ at any given time.
 > * A __mark__ can only have one __receiver__ mounted at any given time.
->
+> * An entry in the session file should be matching the receiver settings and covering its deploy time.
```

Comment:
should there be a similar requirement for a firmware version?

Reply:
this automatic check was never been introduced , so I will remove it for now and will be added once implemented in the codes